### ISSUE TITLE
username mentions

### ISSUE THREAD

**Issue by trosel** [syphon-org/syphon#595]
Is it possible to mention someone's username or nickname?

I tried typing `@` and also tried tapping on the person's name. Neither seemed to work.

Any plans to implement this?

**Comment by dnisbetjones** [comment]
It's not in the official roadmap (https://syphon.org/roadmap) but I do believe I saw this feature mentioned for addition to future releases.

**Comment by trosel** [comment]
do you mention someone just by typing their name then?

**Comment by trosel** [comment]
Yeah, I think it would be wise to implement some kind of message parsing on the device. Then you could have a setting where users can enter keywords that they would like to be notified of, not just their usernames or nicknames.

The "pop a notification only if I'm mentioned" setting is sort of table stakes for messengers these days. Because most people have group chats.

**Comment by EdGeraghty** [comment]
I don't speak for the project, however there are several of us who've switched to ` 0.2.10 ` as our daily driver so I'm expecting to see a whole bunch of improvements soon.

This is a feature I'd also like to see, but yes it'll need to be done as part of a wider implementation (including message parsing, as you mentioned).

Thanks for using Syphon, it's only upwards from here!
